libs/core/langchain_core/tools/convert.py

```diff
@@ -83,6 +88,9 @@ def tool(
             converted to a tool. Must be provided as a positional argument.
         runnable: Optional runnable to convert to a tool. Must be provided as a
             positional argument.
+        description: Optional description for the tool.
```

Comment:
Could you specify the precedence rules?

What if description and arg schema and parse_docstring are all specified/used